CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [v0.4.1]
+> February 1, 2018
+
+- mongoose 5 support (tests pass, code unchanged)
```

Comment:
Would be nice to mention dropping support for node < 4 and mongodb < 3

Reply:
Added a deprecated warning - the code should still be valid